pkg/project/common/aemw

```diff
@@ -124,4 +124,5 @@ export MSYS2_ENV_CONV_EXCL="*"
 # Execute AEM Compose CLI
 # =======================
 
-aem "$@"
+AEM=$(whereis aem | awk '{print $2}')
```

Comment:
In tf aemc this is run without issues. In vscode ext it also is doable to run it without amendment. Please revert this